jwst/skymatch/skymatch.py

```diff
@@ -382,24 +380,26 @@ def match(images, skymethod='global+match', match_down=True, subtract=False):
 
 
 def _apply_sky(images, sky_deltas, do_global, do_skysub, show_old):
+    img_type = ['Image', 'Group']
```

Comment:
I would add a comment here,

```
# The following to be indexed by boolean is_group
img_type = ['Image', 'Group']
```
to make this clear.

Reply:
Done

Reply:
Done again